packages/client/src/workqueue/reducer.ts

```diff
@@ -129,6 +133,84 @@ export function updateRegistrarWorkqueue(
   }
 }
 
+async function getFilteredDeclarations(
+  workqueue: IWorkqueue,
+  getState: () => IStoreState
+): Promise<{
+  currentlyDownloadedDeclarations: IDeclaration[]
+  unassignedDeclarations: IDeclaration[]
+}> {
+  const state = getState()
+  const scope = getScope(state)
+  const savedDeclarations = state.declarationsState.declarations
+
+  const workqueueDeclarations = Object.entries(workqueue.data).flatMap(
+    (queryData) => {
+      return queryData[1].results
+    }
+  ) as Array<GQLEventSearchSet | null>
+
+  let [currentlyDownloadedDeclarations, unassignedDeclarations]: [
+    IDeclaration[],
+    IDeclaration[]
+  ] = [[], []]
```

Comment:
Can we remove this please? From what I could see, we are only assigning the values once